net/netclient/Makefile

```diff
@@ -0,0 +1,62 @@
+# Copyright 2024 Stan Grishin (stangri@melmac.ca)
+# This is free software, licensed under the Apache-2.0 License.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=netclient
+PKG_VERSION:=0.21.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/gravitl/netclient/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=6e8017e1e89530d836a35658d778196d801cd634569c3827858631d9f756ac31
+
+PKG_MAINTAINER:=Stan Grishin <stangri@melmac.ca>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE.txt
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_FLAGS:=no-mips16
+
+GO_PKG:=github.com/gravitl/netclient
+GO_PKG_BUILD_PKG:= \
+	github.com/gravitl/netclient
+GO_PKG_LDFLAGS_X:=\
+	main.version=$(PKG_VERSION)-$(PKG_RELEASE)
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/netclient
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=netclient
+  URL:=https://docs.openwrt.melmac.net/netclient/
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Build/Compile
+  $(call GoPackage/Build/Compile)
+endef
```

Comment:
Isnt this default?

Reply:
I wasn't sure, but based on your feedback I've tried building with these lines deleted and it worked fine. Do you recommend I remove those?

Reply:
You wasnt sure, but upstream is sure about this. :) Please in the future, try to avoid doing something like this. In your Makefile, there are some includes, etc. Try to look at them, what they are doing. You can not write or use something, which you dont understand. Thats the same in the presentation. You dont have in slide something which you dont understand.

You have here something, which is already default, no need to have in the Makefile - lets call the defaults! 

More details:
https://github.com/openwrt/packages/blob/620afc1806d0a2196a4280b68f93990443b4f4e5/lang/golang/golang-package.mk#L299